### PR TITLE
Make the Body and About of Portfolio

### PR DESCRIPTION
**Pull request by VictorOmarT**
Task completed in this PR:

making Body and About 
making sure that the Linters works
update the live demo on README

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -33,10 +34,83 @@ <h3>I’am a Software Developer</h3>
...
+                industry's standard</p>
+            <div class="aling">
+                <img src="pictures/translu.png" alt="">
+                <img src="pictures/translu1.png" alt="">
+                <img src="pictures/translu2.png" alt="">
+            </div>
+            <div class="oragan">
+                <button class="button">See Project</button>
+            </div>
+        </section>
```
Comment (voscarmv) [inline index.html]: - [x] According to the Figma design, projects are supposed to have either a button or a link to `View Project`. Please add these to your projects.
- [x] Please use `<ul>` and `<li>` to properly list the technologies used in each project.
- [x] Please mind proper HTML indentation and delete unnecessary blank lines in your whole document to keep with the best HTML practices. Use 2-space indentation and only use blank lines to separate significant sections of code.

File: main.css [hunk 2]
```diff
@@ -78,3 +78,109 @@ p {
...
+  background-image: url("pictures/ImgPlaceholder2.png");
+  background-repeat: no-repeat;
+  margin-top: 20px;
+  margin-bottom: 60px;
+}
+
+h6 {
+  font-size: 32px;
+  margin-bottom: 0%;
+  margin-left: 20px;
```
Comment (voscarmv) [inline main.css]: - [x] Please keep your code DRY by creating a class for redundant CSS attributes. In this case, `margin-left: 20px` appears 6 times. You may use this tool to check for CSS redundancy in the future: https://courses.cs.washington.edu/courses/cse154/20su/resources/assets/css-redundancy-checker/index.html
image

**Review by voscarmv — CHANGES_REQUESTED** [review]
Hi @VictorOmarT , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- Please check out my comments on best HTML/CSS practices.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Adding a screenshot to your Readme file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -31,14 +31,82 @@ <h3>I’am a Software Developer</h3>
...
+        <section class="activ-g">
+            <h6 class="en">Profesional Art<br>Printing Data <br></h6>
+            <p class="text2">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups
+                required. has been the <br>
+                industry's standard</p>
+            <div class="en">
+                <img src="pictures/translu.png" alt="">
+                <img src="pictures/translu1.png" alt="">
+                <img src="pictures/translu2.png" alt="">
+            </div>
```
Comment (voscarmv) [inline index.html]: - [x] We are still not using proper `<ul>` and `<li>` elements to list the technologies here. Please fix this by using the proper list tags in this and the other projects.

File: index.html [hunk 4]
```diff
@@ -31,14 +31,82 @@ <h3>I’am a Software Developer</h3>
...
+        <h5>Multi-Post Stories</h5>
+        <p class="par-black">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's
+            standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <div>
+            <img src="pictures/Tag.png" alt="">
+            <img src="pictures/Tag(1).png" alt="">
+            <img src="pictures/Tag(2).png" alt="">
+            <img src="pictures/Tag(3).png" alt="">
+        </div>
```
Comment (voscarmv) [inline index.html]: - [ ] This project is still missing a `View Project` `<a>` link or `<burron>`. It's also missing a proper `<ul>` `<li>` list of technologies. Please fix it.

File: index.html [hunk 5]
```diff
@@ -31,14 +31,82 @@ <h3>I’am a Software Developer</h3>
...
+        <p class="endt">Skills</p>
+        <ul class="enlase">
+            <li><a href="#">Codekit</a></li>
+            <li><a href="#">Github</a></li>
+            <li><a href="#">Codepen</a></li>
+            <li><a href="#">Gitlab</a></li>
+            <li><a href="#">Terminal</a></li>
+        </ul>
+        <a href="https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27">View Project</a>
+    </section>
```
Comment (voscarmv) [inline index.html]: - [x] As mentioned earlier, please use proper `2-space indentation` for your HTML. You may change this value in your Visual Studio Code settings. Check out this youtube video on how to do this: https://www.youtube.com/watch?v=m6gUdoRIRz8

**Review by voscarmv — CHANGES_REQUESTED** [review]
Hi @VictorOmarT , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- I noticed you marked most of my requested changes as complete, but did not really apply them. Let's please go back and try to complete all the mandatory requirements for this challenge!

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Adding a screenshot to your README file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by o-rashad — APPROVED** [review]
Hi @VictorOmarT , 

# Approved 🥳 

Good Job implementing the previous reviewer required changes 👏🏼 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Changes 

- [ ] It would be awesome if you would Implement the `View Project` link that was required by the previous reviewer -_and you nicely added it to the end of your page_- to each `See project` button you have with an effect of your choice like `hover` 👍🏼 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**